examples/kafkaconnect/v1beta1/connector-nokafka.yaml

```diff
@@ -0,0 +1,273 @@
+kind: Bucket
+metadata:
+  annotations:
+    meta.upbound.io/example-id: kafkaconnect/v1beta1/connector
+  labels:
+    testing.upbound.io/example-name: connector-nokafka
+  name: ${Rand.RFC1123Subdomain}
+spec:
+  forProvider:
+    region: us-east-2
+    forceDestroy: true
+
+---
+apiVersion: s3.aws.upbound.io/v1beta1
+kind: Object
+metadata:
+  annotations:
+    meta.upbound.io/example-id: kafkaconnect/v1beta1/customplugin
+    uptest.upbound.io/disable-import: "true"
+  labels:
+    testing.upbound.io/example-name: connector-nokafka
+  name: example
+spec:
+  forProvider:
+    bucketSelector:
+      matchLabels:
+        testing.upbound.io/example-name: connector-nokafka
+    # An actual zip file with no data in it
+    contentBase64: >
+      UEsDBAoAAAAAAPpkXlcAAAAAAAAAAAAAAAAFABwAZW1wdHlVVAkAA4gGQGWIBkBldXgLAAEE6AMAAAToAwAAUEsBAh4DCgAAAAAA+mReVwAAAAAAAAAAAAAAAAUAGAAAAAAAAAAAALSBAAAAAGVtcHR5VVQFAAOIBkBldXgLAAEE6AMAAAToAwAAUEsFBgAAAAABAAEASwAAAD8AAAAAAA==
+    contentType: application/zip
+    key: empty.zip
+    region: us-east-2
+
+---
+apiVersion: ec2.aws.upbound.io/v1beta1
+kind: SecurityGroup
+metadata:
+  annotations:
+    meta.upbound.io/example-id: kafkaconnect/v1beta1/connector
+  labels:
+    testing.upbound.io/example-name: connector-nokafka
+  name: sg
+spec:
+  forProvider:
+    region: us-east-2
+    vpcIdSelector:
+      matchLabels:
+        testing.upbound.io/example-name: connector-nokafka
+
+---
+apiVersion: ec2.aws.upbound.io/v1beta1
+kind: Subnet
+metadata:
+  annotations:
+    meta.upbound.io/example-id: kafkaconnect/v1beta1/connector
+  labels:
+    testing.upbound.io/example-name: connector-nokafka
+  name: subnet-az1
+spec:
+  forProvider:
+    availabilityZone: us-east-2a
+    cidrBlock: 192.168.0.0/24
+    region: us-east-2
+    vpcIdSelector:
+      matchLabels:
+        testing.upbound.io/example-name: connector-nokafka
+
+---
+apiVersion: ec2.aws.upbound.io/v1beta1
+kind: Subnet
+metadata:
+  annotations:
+    meta.upbound.io/example-id: kafkaconnect/v1beta1/connector
+  labels:
+    testing.upbound.io/example-name: connector-nokafka
+  name: subnet-az2
+spec:
+  forProvider:
+    availabilityZone: us-east-2b
+    cidrBlock: 192.168.1.0/24
+    region: us-east-2
+    vpcIdSelector:
+      matchLabels:
+        testing.upbound.io/example-name: connector-nokafka
+
+---
+apiVersion: ec2.aws.upbound.io/v1beta1
+kind: VPC
+metadata:
+  annotations:
+    meta.upbound.io/example-id: kafkaconnect/v1beta1/connector
+  labels:
+    testing.upbound.io/example-name: connector-nokafka
+  name: vpc
+spec:
+  forProvider:
+    cidrBlock: 192.168.0.0/22
+    region: us-east-2
+
+---
+apiVersion: cloudwatchlogs.aws.upbound.io/v1beta1
+kind: Group
+metadata:
+  annotations:
+    meta.upbound.io/example-id: kafkaconnect/v1beta1/connector
+    crossplane.io/external-name: ${Rand.RFC1123Subdomain}
+  labels:
+    testing.upbound.io/example-name: connector-nokafka
+  name: test
+spec:
+  deletionPolicy: Orphan
```

Comment:
Why do we set `deletionPolicy: Orphan`?

Reply:
Leftover from manual testing where I wanted to see the actual logs to get the connector working. Will remove (from both examples)